website/addons/wiki/static/pagedown-ace/Markdown.Editor.js

```diff
@@ -1704,9 +1721,13 @@ var Range = ace.require('ace/range').Range;
 
             buttons.redo = makeButton("wmd-redo-button", getStringAndKey("redo"), "-220px", null);
             buttons.redo.execute = function (manager) { inputBox.session.getUndoManager().redo(); };
+
             makeSpacer(4);
             makeCheckBox("wmd-autocom-toggle", "autocom", "-240px", "Autocomplete");
 
+            makeSpacer(5);
+            buttons.help = makeHelpButton("wmd-help-button", getStringAndKey("help"), "-240px");

```

Comment:
Steve's correct. When this tries to get the hotkey for `help` without `help` being in `keyStrokes`, it raises a `TypeError` somewhere.